pkg/services/dashboards/service/dashboard_service.go

```diff
@@ -745,7 +749,7 @@ func (dr *DashboardServiceImpl) Kind() string { return entity.StandardKindDashbo
 
 func (dr *DashboardServiceImpl) CleanUpDeletedDashboards(ctx context.Context) (int64, error) {
 	var deletedDashboardsCount int64
-	deletedDashboards, err := dr.dashboardStore.GetSoftDeletedDashboards(ctx)
+	deletedDashboards, err := dr.dashboardStore.GetSoftDeletedDashboardsByTime(ctx, remainingTrashDays)
```

Comment:
`ByTime` implies the parameter will be a time which is not the case. I would change the name to something like `getSoftDeletedExpiredDashboards`